Formula/zeek.rb

```diff
@@ -1,9 +1,10 @@
 class Zeek < Formula
   desc "Network security monitor"
-  homepage "https://www.zeek.org"
+  homepage "https://zeek.org"
   url "https://github.com/zeek/zeek.git",
       :tag      => "v3.1.4",
       :revision => "6747d93745a00d24a376d4633280a47a60961e10"
+  revision 1
```

Comment:
This shouldn't be needed

Reply:
Ok. I've removed the two Zeek-related commits.

Out of curiosity, why isn't it necessary this time? The revision bump was needed on the last two PRs.

Reply:
Because the version changed. Which isn't the case here

Reply:
Maybe the revision bump is still needed after all?

```
==> FAILED
==> Testing zeek
/usr/bin/sandbox-exec -f /private/tmp/homebrew20200730-29991-pnuwhs.sb ruby -W0 -I $LOAD_PATH -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/zeek.rb --retry --verbose
==> /usr/local/Cellar/zeek/3.1.5/bin/zeek --version
dyld: Library not loaded: /usr/local/opt/caf/lib/libcaf_core.0.17.5.dylib
  Referenced from: /usr/local/Cellar/zeek/3.1.5/bin/zeek
  Reason: image not found
```

Zeek still looks for CAF 0.17.5. Probably, because the CAF version was ill-reported in the original tag.